dags/blocksec_plugin/contract_interaction_operator.py

```diff
@@ -69,9 +72,8 @@ def execute(self, context):
                                maxPriorityFeePerGas=self.web3.eth.max_priority_fee,
                                gas = self.gas
                               ))
-        # if not self.confirm_success(raw_txn):
-        #     raise ValueError("Transaction failed to confirm")
+
         signed_txn = self.web3.eth.account.signTransaction(raw_txn, self.wallet.private_key)
         transaction_hash = self.web3.eth.sendRawTransaction(signed_txn.rawTransaction)
-        print(f"Txn hash: {transaction_hash.hex()}")
-        return str(transaction_hash.hex())
+
+        return EthereumTransactionConfirmationSensor(transaction_hash=transaction_hash).poke(context)
```

Comment:
If you plan to try this approach, you will need to also add `confirmations=self.confirmations` into this, otherwise it'll always be the default number of confirmations